src/Microsoft.AspNetCore.Server.Kestrel/Microsoft.AspNetCore.Server.Kestrel.csproj

```diff
@@ -4,19 +4,22 @@
 
   <PropertyGroup>
     <Description>ASP.NET Core Kestrel cross-platform web server.</Description>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;netcoreapp2.0</TargetFrameworks>
```

Comment:
Just move this to the Core project. Currently only the "meta" project only contains WebHostBuilderKestrelExtensions.